ops/demo/_data.tf

```diff
@@ -172,3 +172,8 @@ data "azurerm_key_vault_secret" "datahub_url" {
   name         = "datahub-url-test"
   key_vault_id = data.azurerm_key_vault.global.id
 }
+
+data "azurerm_key_vault_secret" "datahub_signing_key" {
+  name         = "datahub-signing-key-test"
+  key_vault_id = data.azurerm_key_vault.global.id
```

Comment:
sometimes we have
`key_vault_id = data.azurerm_key_vault.global.id`
and sometimes we use
`key_vault_id = data.azurerm_key_vault.sr_global.id`
is there a difference between these two vaults or is this a typo?

Reply:
oh I see, it depends on how we define the key vault in the tf file
e.g. dev4 uses sr_global: https://github.com/CDCgov/prime-simplereport/blob/3598f5a36d42b8aa0dc5edac7db7b2909fdd923d/ops/dev4/_data.tf#L45
nvm!

Reply:
Not a typo, just a little human inconsistency. The key vault ID is declared at the top of these files, and while the identifiers may have different names they still point to the same resource.

https://github.com/CDCgov/prime-simplereport/blob/3598f5a36d42b8aa0dc5edac7db7b2909fdd923d/ops/demo/_data.tf#L44-L48

https://github.com/CDCgov/prime-simplereport/blob/3598f5a36d42b8aa0dc5edac7db7b2909fdd923d/ops/dev/_data.tf#L44-L48